test/runnable/inline.d

```diff
@@ -736,21 +736,50 @@ void test14753(string) { }
 
 /**********************************/
 
-struct S14975 {
+struct S14975
+{
```

Comment:
I don't believe that { } style in the test suite matters in the slightest, and so object to reformatting them as being pointless churn.